src/container.ts

```diff
@@ -64,10 +62,11 @@ container.register({
 });
 
 // Redis server settings.
-const DEFAULT_TOKEN_DURATION_IN_SECONDS = 3600;
-const tokenDurationInSeconds =
-  parseInt(process.env.TOKEN_DURATION_IN_SECONDS) ||
-  DEFAULT_TOKEN_DURATION_IN_SECONDS;
+const DEFAULT_TOKEN_DURATION_IN_SECONDS = "3600";
+const tokenDurationInSeconds: number = parseInt(
```

Comment:
```ts
const tokenDurationInSeconds: number = process.env.TOKEN_DURATION_IN_SECONDS ?
 parseInt(process.env.TOKEN_DURATION_IN_SECONDS, 10) : DEFAULT_TOKEN_DURATION_IN_SECONDS
```

Reply:
I would consider it an error to set TOKEN_DURATION_IN_SECONDS to a non-number string